src/clims/models/extensible.py

```diff
@@ -1,10 +1,44 @@
 from __future__ import absolute_import, print_function
 
 from django.db import models
-from sentry.db.models import (Model, sane_repr)
+from sentry.db.models import (Model, sane_repr, FlexibleForeignKey)
 import json
 
 
+class ExtensibleModel(Model):
+    """
+    Base class for extensible models
+    """
+    __core__ = True
```

Comment:
Good question. The `__core__` flag is only being used to decide if a table should be exported or not. It makes sense that this data should not be exported as it's not settings data. I'll set it to `False`. I've also created a JIRA ticket to go through this everywhere `CLIMS-243` (next milestone).

Reply:
I would then say it makes sense to have the type info core, even if it's defined in a plugin. Of course I would prefer that we would set this data only via plugins, but it may be good to be able to export everything that is a "setting".